actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -313,6 +313,13 @@ def tag_option(key, value, escape)
             value = escape ? safe_join(value, " ") : value.join(" ")
           when Regexp
             value = escape ? ERB::Util.unwrapped_html_escape(value.source) : value.source
+          when IPAddr
```

Comment:
Should this possibly be handled in the `IPAddr#to_s` flow?

Reply:
@mikeheft Thanks for your comment.

I think the changing behavior of `IPAddr#to_s` has breaking compatibility.
For example, it will be unintended behavior to get an address with prefix using `IPAddr#to_s` and `IPAddr#prefix`.

The sample code is as follows.
```ruby
# `IPAddr#to_s` return address without prefix (current behavior of ipaddr gem v1.2.5)
addr = IPAddr.new("192.0.2.0/24")

addr.to_s
#=> "192.0.2.0"

str = format("%s/%s", addr.to_s, addr.prefix)
assert_equal("192.0.2.0/24", str)
#=> true
```

If `IPAddr#to_s` changes the behavior, it will fail in this use case.
```ruby
# if `IPAddr#to_s` return address with prefix
addr.to_s
#=> "192.0.2.0/24"

str = format("%s/%s", addr.to_s, addr.prefix)
assert_equal("192.0.2.0/24", str)
#=> false
# str => "192.0.2.0/24/24"
```

Reply:
💯 Just throwing it out there. It would definitely require a little more work in the IPAddr module. My thoughts were that it would negate the need to use the `#format` method to get them both. And instead add something like an `#ip_address` method that would return what is currently returned by `#to_s`. 
Regardless, you are 💯 , it'd be a breaking change to the IPAddr module and would need lots more discussion there